src/rules/declaration-block-no-ignored-properties/README.md

```diff
@@ -0,0 +1,53 @@
+# declaration-block-no-ignored-properties
+
+Disallow properties has no effect.
+
+```css
+a { display: inline; width: 100px; }
+/**                        ↑
+ *                  This property */
+```
+
+The following patterns are considered warnings:
+
+```css
+a { display: inline; width: 100px; }
+```
+
+```css
+a { display: inline; height: 100px; }
+```
+
+```css
+a { display: inline; margin: 10px; }
+```
+
+```css
+a { display: inline-block; float: left; }
+```
+
+```css
+a { display: block; vertical-align: baseline; }
+```
+
```

Comment:
I think that to really explain this rule it would help to add very brief explanations after each example, e.g. "`display: inline` causes `width` to be ignored." & "`display: block` causes `vertical-align` to be ignored."